src/main/java/core/basesyntax/service/impl/WriterServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WriterService;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+
+public class WriterServiceImpl implements WriterService {
+    @Override
+    public void write(String toFileName, String report) {
+        File file = new File(toFileName);
+        try {
+            if (file.exists()) {
+                Files.write(file.toPath(), report.getBytes(), StandardOpenOption.APPEND);
+            } else {
+                file.createNewFile();
+                Files.write(file.toPath(), report.getBytes(), StandardOpenOption.APPEND);
+            }
```

Comment:
personally, I'm not sure about this, maybe you should go easier way like just writing to a file whether it exists or not